client/mig/main.go

```diff
@@ -94,6 +99,8 @@ func main() {
 	fs.StringVar(&show, "show", "found", "type of results to show")
 	fs.StringVar(&render, "render", "text", "results rendering mode")
 	fs.StringVar(&target, "t", fmt.Sprintf("status='%s' AND mode='daemon'", mig.AgtStatusOnline), "action target")
+	fs.StringVar(&targetfound, "target-foundanything", "", "Limit targets to agents that returned positive results")
+	fs.StringVar(&targetnotfound, "target-foundnothing", "", "Limit targets to agents that did not returned positive results")
```

Comment:
These explanations are vague. What about `targets agents that have found results in a previous action. ex: -target-found 123456`.